packages/prompts/package.json

```diff
@@ -57,7 +57,6 @@
     "sisteransi": "^1.0.5"
   },
   "devDependencies": {
-    "unbuild": "^1.1.1",
     "is-unicode-supported": "^1.3.0"
```

Comment:
Unrelated to this PR, but should this really be a dev dep?

Reply:
I plan on addressing that too. It should definitely not be in `bundledDependencies`.